backend/app/minio/utils.py

```
@@ -0,0 +1,116 @@
+    return f"http://localhost:9000/{BUCKET_NAME}/{file_name}"
+
+
+def validate_file_type(content_type: str, upload_type: str) -> None:
+    if upload_type in ["profile", PRODUCT_PHOTO]:
+        if content_type not in ALLOWED_IMAGE_TYPES:
+            raise HTTPException(
+                status_code=400,
+                detail=f"Invalid file type: {content_type}. Only JPEG and PNG images are allowed.",
+            )
+    elif upload_type == PRODUCT_VIDEO:
+        if content_type not in ALLOWED_VIDEO_TYPES:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)